lib/config.ts

```diff
@@ -261,6 +261,9 @@ export type Config = {
     scihub: {
         host?: string;
     };
+    skeb: {
+        bearer_token?: string;
```

Comment:
```suggestion
        bearerToken?: string;
```
[`camelCase`](https://docs.rsshub.app/joinus/advanced/script-standard#naming)